src/operator/nn/dropout-inl.h

```diff
@@ -492,7 +492,7 @@ class DropoutOp {
 #endif  // MXNET_USE_CUDNN_DROPOUT
 };  // class DropoutOp
 
-static OpStatePtr CreateDropoutState(const nnvm::NodeAttrs &attrs,
+inline OpStatePtr CreateDropoutState(const nnvm::NodeAttrs &attrs,
```

Comment:
Is this a duplicate fix with #14900 ? Could you merge your changes in that PR into this one here?

Reply:
Since #14900 is merged now, please do a rebase when you have time to do so, thanks